src/main.rs

```diff
@@ -22,7 +22,7 @@ fn validate_grading_system() {
 
 /// Calculate the grade
 fn calculate_grade(carrots: i32, nuts: i32, seeds: i32) -> i32 {
-    todo!()
+    return (carrots + nuts) * (carrots + nuts + seeds);
```

Comment:
Remember, we don't need to add the `return` keyword, since it's implicitly added at the end of functions :)